app/src/main/res/layout/controller_locked.xml

```diff
@@ -18,36 +20,46 @@
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
 
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/unlockContainer"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/ic_launcher_background"
+    android:layout_centerInParent="true"
+    android:background="@color/colorPrimary"
+    android:gravity="center"
     android:orientation="vertical">
 
-    <ImageView
-        android:id="@+id/appLogo"
-        android:layout_width="92dp"
-        android:layout_height="92dp"
-        android:layout_centerInParent="true"
-        android:scaleType="center"
-        android:src="@drawable/ic_launcher_foreground" />
+    <FrameLayout
+        android:id="@+id/appLogoContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal">
+
+        <ImageView
+            android:id="@+id/appLogo"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/nc_app_name"
+            android:src="@drawable/ic_logo" />
 
-    <ImageView
-        android:layout_width="24dp"
-        android:layout_height="24dp"
-        android:layout_alignTop="@id/appLogo"
-        android:layout_alignEnd="@id/appLogo"
-        android:src="@drawable/ic_lock_white_24px" />
+        <ImageView
+            android:layout_width="32dp"
+            android:layout_height="32dp"
+            android:layout_gravity="center"
+            android:src="@drawable/ic_lock_white_24px" />
+
+    </FrameLayout>
 
     <TextView
         android:id="@+id/unlockTextView"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_below="@id/appLogo"
-        android:layout_centerHorizontal="true"
-        android:layout_marginTop="@dimen/margin_between_elements"
-        android:padding="16dp"
+        android:layout_gravity="center_horizontal"
+        android:paddingTop="16dp"
+        android:paddingBottom="56dp"
         android:text="@string/nc_locked"
         android:textAlignment="center"
-        android:textColor="@color/fg_inverse" />
-</RelativeLayout>
+        android:textColor="@color/white"
```

Comment:
since the backgroups is color/colorPrimary, this here should use the textColorOnPrimaryBackground again?
```suggestion
        android:textColor="@color/textColorOnPrimaryBackground"
```

Reply:
DONE https://github.com/nextcloud/talk-android/pull/1147/commits/c2ac2705a3dfef72fa06a8de41cb8595bdb65daa